src/server/server_family.cc

```diff
@@ -2040,6 +2040,7 @@ void ServerFamily::Info(CmdArgList args, ConnectionContext* cntx) {
     append("blocked_on_interpreter", m.coordinator_stats.blocked_on_interpreter);
     append("ram_hits", m.events.ram_hits);
     append("ram_misses", m.events.ram_misses);
+    append("script_erros", script_mgr_->script_errors());
```

Comment:
we already have "script_errors" in "info all".  see `should_enter("ERRORSTATS", true)` section.
the goal is actually to expose script errors via /metrics